hydro_test_local/src/local/futures.rs

```diff
@@ -0,0 +1,97 @@
+use std::time::Duration;
+
+use dfir_rs::tokio;
+use dfir_rs::tokio::sync::mpsc::UnboundedSender;
+use hydro_lang::deploy::SingleProcessGraph;
+use hydro_lang::dfir_rs::scheduled::graph::Dfir;
+use hydro_lang::*;
+use stageleft::{q, Quoted, RuntimeData};
+
+#[stageleft::entry]
+pub fn unordered<'a>(
+    flow: FlowBuilder<'a>,
+    output: RuntimeData<&'a UnboundedSender<u32>>,
+) -> impl Quoted<'a, Dfir<'a>> {
+    let process = flow.process::<()>();
+
+    process
+        .source_iter(q!([2, 3, 1, 9, 6, 5, 4, 7, 8]))
+        .map(q!(|x| async move {
+            // tokio::time::sleep works, import then just sleep does not, unsure why
```

Comment:
Very strange... this would indicate that something is wrong with the `stageleft` logic... But irrelevant to this PR so okay for now.